olive/data/template.py

```diff
@@ -123,3 +123,141 @@ def raw_data_config_template(
             }
         ),
     )
+
+
+def transformers_dummy_data_config_template(
+    model_name,
```

Comment:
What's the use of these templates?

Reply:
API purpose. In olive, we can directly call this function to create a dc without json config.

Reply:
I don't see any use of this function currently.

Reply:
Just removed. Anyone can add if they need it like we did for `dummy data config, huggingface data config`.